### PR TITLE
Add support for Symfony 7.0

### PR DESCRIPTION
**Pull request by jerowork**
# Description
Update `composer.json` to support Symfony `^7.0`.

### CODE REVIEW

**Comment by ruudk** [comment]
@Yozhef could you approve the workflow, merge it, and tag it 🙏 Thanks

**Review by Yozhef — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -16,11 +16,11 @@
 
     "require": {
         "php":                   "^7.4|^8.0",
-        "symfony/css-selector":  "^4.4|^5.0|^6.0"
+        "symfony/css-selector":  "^4.4|^5.0|^6.0|^7.0"
```
Comment (Yozhef) [inline composer.json:19]: Please add to CI process to run Symfony version 7.0

**Review by Yozhef — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (jerowork) [reply composer.json:19]: Sharp, added 7.0 to be tested for PHP8.2

**Comment by jerowork** [comment]
Thanks @Yozhef 🎉

**Comment by Yozhef** [comment]
@jerowork thank you very much for the contribution and development - sorry for taking so long to watch

**Pull request merged**